client/landing/gutenboarding/steps.ts

```diff
@@ -8,6 +8,7 @@ export enum Step {
 	DesignSelection = '/design',
 	PageSelection = '/pages',
 	Signup = '/signup',
+	SiteLoading = '/siteLoading',
```

Comment:
A "loading" transitional state doesn't seem like something that deserves a route. How about "create site," which captures the intent when we hit this route?

```suggestion
	CreateSite = '/create-site',
```

cc: @razvanpapadopol